src/coreclr/hosts/unixcoreruncommon/coreruncommon.cpp

```diff
@@ -232,6 +239,15 @@ int ExecuteManagedAssembly(
     std::string tpaList;
     AddFilesFromDirectoryToTpaList(clrFilesAbsolutePath, tpaList);
 
+#if defined(__LINUX__)
+    void* coreclrTPLib = dlopen(coreClrTPproviderDllPath.c_str(),RTLD_NOW | RTLD_GLOBAL);
+    if (coreclrTPLib == nullptr)
+    {
+        char *error_msg = dlerror();
+        fprintf(stderr, "Event Tracing is disabled.\n dlopen failed to load tracepoint provider:%s, with error:%s\n",coreClrTpProviderDll,error_msg);
```

Comment:
I don't think that we should be logging any errors if the tracepoint provider can't be loaded.  This is considered to be a "normal scenario" where LTTng is not installed on the machine.